lib/pure/strutils.nim

```diff
@@ -1330,18 +1330,35 @@ proc join*[T: not string](a: openArray[T], sep: string = ""): string {.
     add(result, $x)
 
 type
-  SkipTable = array[char, int]
+  SkipTable* = array[char, int]
```

Comment:
Why is this now exported?

Reply:
This type should be accessible from other modules.
And how else can I call a new find function if this type is not exported?